### PR TITLE
Contact Form: Adding non minified JS file to ensure there is a choice of non-minified and minified

### PR DESCRIPTION
**Pull request by coder-karen**
Fixes https://github.com/Automattic/vulcan/issues/316

## Proposed changes:

* ~This PR adds the file path to the non-minified version of `tinymce-plugin-form-button.js`, into `Assets::get_file_url_for_environment` within `mce_external_plugins`.~ Previously both minified versions (located at `jetpack_vendor/automattic/jetpack-forms/dist/contact-form/js/tinymce-plugin-form-button.js`) had been added to `Assets::get_file_url_for_environment`, as a quick fix to prevent an issue caused by calling the minified JS file using `.min.js`, however that file path does not exist in the package version of the contact form.
* After a discussion on Slack, this PR is now using `plugins_url` and only using the minified file version. Slack ref: p1715614401962229-slack-CBG1CP4EN
* Other options included ensuring the minified version name was changed to include `min`, however that didn't seem necessary.

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [x] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [x] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

https://github.com/Automattic/vulcan/issues/316

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

* To test, apply this PR and then install and activate the Classic Editor plugin.
* Build the assets with `jetpack build packages/forms --production`
* Create a new post draft, and use the 'Add Contact Form' button to add a new form (or open the editor for a post that already has a form added using this method). It should add with no issues.
* There should also be no issues on WoA sites, nor simple sites, with this patch.

### CODE REVIEW

**Review by monsieur-z — DISMISSED** [review]
It worked fine with self-hosted, simple, and WoA sites.

**Comment by coder-karen** [comment]
Thanks @monsieur-z ! I've completely changed the code now after a comment on Slack - p1715614401962229-slack-CBG1CP4EN - so this'll need re-review.

File: projects/packages/forms/src/contact-form/class-editor-view.php [hunk 1]
```diff
@@ -59,10 +60,8 @@ public static function grunion_media_button() {
 	 * @return array
 	 */
 	public static function mce_external_plugins( $plugin_array ) {
-		$plugin_array['grunion_form'] = Assets::get_file_url_for_environment(
-			'jetpack_vendor/automattic/jetpack-forms/dist/contact-form/js/tinymce-plugin-form-button.js',
-			'jetpack_vendor/automattic/jetpack-forms/dist/contact-form/js/tinymce-plugin-form-button.js'
-		);
+		$plugin_array['grunion_form'] = plugins_url( 'jetpack_vendor/automattic/jetpack-forms/dist/contact-form/js/tinymce-plugin-form-button.js', Jetpack_Constants::get_constant( 'JETPACK__PLUGIN_FILE' ) );
```
Comment (fgiannar) [inline projects/packages/forms/src/contact-form/class-editor-view.php]: How come we are not doing something like: `plugins_url( '../../dist/contact-form/js/tinymce-plugin-form-button.js', __FILE__ );` so we don't have to rely on the `JETPACK__PLUGIN_FILE` constant defined only by the Jetpack plugin?

Comment (coder-karen) [reply projects/packages/forms/src/contact-form/class-editor-view.php]: I can only assume I was inputting the path wrongly initially as that was my first choice, but on testing it all again now it works well. Thank you for flagging that.

**Review by monsieur-z — APPROVED** [review]
Still works!

**Pull request merged**